0000-template.md

```diff
@@ -5,26 +5,23 @@ rfc pr: (leave this empty)
 related issue: (tracking issue number)
 ---
 
-<!-- replace the blockquoted sections with your content -->
 # Summary
 
 > Brief description of the feature.
 
+# README
+
+> Work backwards from the README: pretend that this feature is already
+> implemented and write the README section. It will help you think about your
+> proposal from a user experience point of view. If a README format does not
+> make sense for your proposal, write a "Press Release" which announces your new
```

Comment:
I worry about people reading this and not quite understanding what we are looking for. Can we link an example specifically for the "press release" format?

I also don't love the idea of their being 2 potential formats, README or PR depending on the feature. I'm kinda in favor of just picking "press release" with some predetermined sections that we define to help guide RFC authors.

Reply:
Let's give it a bit of time if that's okay with you and see if we need to make this a bit more concrete. We will gather some examples so we can reference.